manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -131,6 +131,7 @@ properties:
     additional_frontend_config: |
       capture response header strict-transport-security len 128
       http-response add-header Strict-Transport-Security max-age=31536000 unless { capture.res.hdr(0) -m found }
+    user_agent_frontend: true
```

Comment:
I commented something similar it in https://github.com/alphagov/paas-haproxy-release/pull/3#issuecomment-229394248